structure_tests/tests/structure_test_tests.sh

```diff
@@ -34,8 +34,7 @@ popd
 
 # Run the debian tests, they should always pass on latest
 envsubst < cloudbuild.yaml.in > cloudbuild.yaml
-gcloud container builds submit . --config=cloudbuild.yaml
-if [ "$?" -gt "0" ]
+if [ "$(gcloud container builds submit . --config=cloudbuild.yaml)" -gt "0" ]
```

Comment:
didn't realize we needed the return value from this command (as well as checking the return code). what this shellcheck is asking us to do is impossible if we need both values, so let's just disable it.